ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -167,42 +169,60 @@ public class BottomSheetController: UIViewController {
         }
     }
 
-    /// Sets the `isHidden` property with a completion handler.
+    /// Changes the `isHidden` state with a completion handler.
```

Comment:
This is super unnecessary but I changed the wording because "sets property" could imply the change will be reflected immediately, but in case of an animated change it will only be reflected when the animation completes...